pandas/core/frame.py

```diff
@@ -1520,6 +1520,28 @@ def to_feather(self, fname):
         from pandas.io.feather_format import to_feather
         to_feather(self, fname)
 
+    def to_parquet(self, fname, engine, compression=None,
+                   **kwargs):
+        """
+        write out the binary parquet for DataFrames
+
+        .. versionadded:: 0.20.0
+
+        Parameters
+        ----------
+        fname : str
+            string file path
```

Comment:
On the other hand, from a Dask perspective it might be nice to one day rely entirely on a `pandas.read_parquet` function for chunk-wise logic.  In this case we would want to hand pandas a file-like object and ask it to get us a few particular row groups from that object.  If inconvenient I don't think we should worry about this use case near-term.  I just thought I'd bring it up.